api/health.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
```

Comment:
Please add a comment attributing this file to the original source and explaining that we needed to copy it to add the authorization metadata.